feat: add filters for gt labelling and gt script-type

jfrer opened the pull request:
closes #68

t11r commented on src/components/workflows/timeline/TimelineFilters.vue:
```diff
@@ -25,6 +25,43 @@ const selectedWorkflows = ref<DropdownOption[]>([])
 
+const labellingOptions = computed(() => {
+  return [
+      ...new Set(workflowsStore.gt.map(gt => 
```
This could use a comment explaining what is happening here.

jfrer replied:
Added a comment in b4f790d.

t11r commented on src/components/workflows/timeline/TimelineFilters.vue:
```diff
@@ -67,13 +114,23 @@ const selectGTs = () => {
     const gt = workflowsStore.getGtById(value)
     if(!gt) return false
-    return hasSomeSelectedProcessor(gt) && hasSomeSelectedDateRange(gt) && hasSomeSelectedWorkflow(gt)
+    return hasSomeSelectedLabelling(gt) && hasSomeSelectedScriptType(gt) && hasSomeSelectedProcessor(gt) && hasSomeSelectedDateRange(gt) && hasSomeSelectedWorkflow(gt)
```
This should be split into multiple lines.

jfrer replied:
Done in 0468788.

t11r commented:
The code itself looks fine, but there a some accessibility concerns:

- All controls should be labelled, and while the checkbox for toggling all filters has an ARIA attribute, a label legible by everyone would not hurt.
- There are many contrast errors, but only few of those have been introduced by this PR.

I would suggest to check with the [WAVE browser extension](https://wave.webaim.org/extension/).

Also, lots of Vue warnings are logged to the console about missing props, extraneous non-props attributes and missing translations. This should be investigated.

jfrer commented:
> The code itself looks fine, but there a some accessibility concerns:
> 
> * All controls should be labelled, and while the checkbox for toggling all filters has an ARIA attribute, a label legible by everyone would not hurt.
> * There are many contrast errors, but only few of those have been introduced by this PR.
> 
> I would suggest to check with the [WAVE browser extension](https://wave.webaim.org/extension/).
> 
> Also, lots of Vue warnings are logged to the console about missing props, extraneous non-props attributes and missing translations. This should be investigated.

Thanks for the extensive review. 

* I added a label to the checkbox for toggling all filters in 9639690. As there is no native functionality/slot for this in the PrimeVue component we use, I had to implement it manually. In this move I implemented a wrapper component to avoid/reduce code duplication.

* The WAVE Tool seems very nice, but I can't find contrast errors that have been introduced by this PR. Can you specify where you found these?
I will resolve the rest of the contrast errors in a seperate PR.

* I also noticed the Vue warnings when I started this project a month ago and got so annoyed by the amount of warnings that I deselected the warnings filter in the console. 😅 I will also try to fix them in a seperate PR.

t11r commented:
>     * The WAVE Tool seems very nice, but I can't find contrast errors that have been introduced by this PR. Can you specify where you found these?

Selected items in the dropdowns have a contrast ratio of 2.55 or lower, while WCAG AA requires at least 4.5 for normal-sized text. In general, the blue is too light to put in on a white or colored background.

jfrer commented:
> Selected items in the dropdowns have a contrast ratio of 2.55 or lower, while WCAG AA requires at least 4.5 for normal-sized text. In general, the blue is too light, especially on a colored background.

Ah, thanks, I see. As we didn't put much thought into coloring yet, these are just default color stylings from the [`lara-light-blue` theme](https://v3.primevue.org/theming/). I will change the theme to a more accessible one when I will work on #91, because I think that a color theme change is a little bit out of scope of this PR. For now, I made the coloring of the "Select all"-Label a variable that will change with the theming.
